src/main/java/at/hannibal2/skyhanni/features/bingo/card/BingoCardTips.kt

```diff
@@ -19,10 +22,24 @@ class BingoCardTips {
 
     private val config get() = SkyHanniMod.feature.event.bingo.bingoCard
 
+    private val patternGroup = RepoPattern.group("bingo.card.tips")
+    private val inventoryPattern by patternGroup.pattern(
+        "card", 
+        "Bingo Card"
+    )
+    private val rewardPattern by patternGroup.pattern(
+        "reward", 
+        "§.§.§7Reward"
```

Comment:
Technically this string always has to start with `§5§o` as this is a minecraft limitation, not controlled by hypixel.